template.js

```diff
@@ -9,71 +9,81 @@ const JSON = require('JSON');
 const logToConsole = require('logToConsole');
```

Comment:
It will be great if you merge your changes with this pull request https://github.com/exponea/gtm-serverside-client-template/pull/2